### ISSUE TITLE
Consider adding a blob create only trigger

### ISSUE THREAD

**Issue by vkurpad** [Azure/azure-webjobs-sdk#1005]
Currently a blob triggered Azure function is invoked each time a blob is created or updated. There are scenarios where a function only needs to know when a new blob is created and it would be great to be able to specify that in the trigger.

**Comment by mathewc** [comment]
Yes, in general it would be useful to allow the function to specify one or more "file watch types" to apply, e.g. delete only, etc.

**Comment by aloysiustany** [comment]
@fabiocav when do you think this would likely be implemented ?

**Comment by mathewc** [comment]
Related to https://github.com/Azure/azure-webjobs-sdk/issues/1005

**Comment by simonech** [comment]
Any news on this?

**Comment by zebarg** [comment]
Any news?

**Comment by m1dst** [comment]
Totally agree this is useful.
